src/plugins/info/info_impl.cpp

```diff
@@ -36,19 +41,26 @@ void InfoImpl::enable()
     // We can't rely on System to request the autopilot_version,
     // so we do it here, anyway.
     _parent->send_autopilot_version_request();
+    _parent->send_flight_information_request();
 
     // We're going to retry until we have the version.
     _parent->add_call_every(
         std::bind(&InfoImpl::request_version_again, this), 1.0f, &_call_every_cookie);
+
+    // We're going periodically ask for the flight information
+    _parent->add_call_every(
+        std::bind(&InfoImpl::request_flight_information, this), 1.0f, &_flight_info_call_every_cookie);
```

Comment:
Do we really have to check every second? We could also check if armed/disarmed has changed and then request it again.

Reply:
@julianoes I have a fix ready for this. Basically check at every disarm. The issue now is that if we decide to do it this way we need to remove arming_time_utc and takeoff_time_utc because they will always be zero. Do you prefer check at every disarm and keep only boot_time and flight_uuid or keep it like it is now and keep the whole FlightInfo structure.

Reply:
So generally something should work when it is merged to MAVSDK. Hence, we either need to implement it on the PX4 side and get it merged, or you need to remove it from the MAVSDK API.

Reply:
@julianoes I'm not sure I understand. What I meant is that they are 0 because you are disarmed when you read those. When you arm they are refreshed but we never ask for the message then because we check only at disarm since that's when the flight_uuid is incremented.

Reply:
Oh got it. Maybe just update it for both changes then.

Reply:
as discussed with @julianoes offline we will check for this message only at disarm and won't read arming_time_utc and takeoff_time_utc. This is maybe something more for the telemetry plugin. I will push it in the next few minutes